joomla/joomla-cms
# [4.0] Fix Upgrade with postgresql- From Joomla v3.9.x to v4.0

Pull Request for Issue #25467.

### Summary of Changes
Use right data type of template styles home column in db.
Synchronize PR #25477 with PR #25484.


### Testing Instructions
Install a copy of Joomla 3.9.10 on postgres.
Apply this PR to your joomla source directory.
Build an update package to your joomla-4.0-dev source directory:
git tag 4.0.0-alpha11-dev
php build/build.php
Update Joomla 3.9.10 with the new update package.
Check if Joomla upgrades successfully


### Expected result



### Actual result



### Documentation Changes Required
